meta-mender-raspberrypi/recipes-bsp/u-boot/patches/experimental/0001-rpi-always-set-fdt_addr-with-firmware-provided-FDT-address.patch

```diff
@@ -0,0 +1,48 @@
+From 226f02f8db76ad7d0431fc10d3f2431c7ce74b06 Mon Sep 17 00:00:00 2001
```

Comment:
I can't see that this patch is listed anywhere. In what context is it to be used?